app/components/home/UserHome/Index.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import AppBar from '../../../redux/containers/dialogs/UserAppBar.js';
+import Body from '../../../redux/containers/UserHome/Body.js';
+import {
+  fetchDocuments,
+  fetchTags
+} from '../../../redux/actions';
+import store from '../../../redux/store';
+import socket from '../../../socket';
+
+// When this client receives the fetch documents event from the server
+socket.on('fetchDocuments', () => {
+  // Fetch documents and tags from server to reflect the change
+  const token = localStorage.getItem('token');
+  fetchDocuments(token, {}, (action) => {
+    store.dispatch(action);
+  });
+
+  fetchTags(token, (action) => {
+    store.dispatch(action);
+  });
+});
+
+class UserHome extends React.Component {
+  // After the component has been loaded
+  componentDidMount() {
+    // Get user token, they can only be here if they are authenticated
+    const token = localStorage.getItem('token');
+
+    // Fetch all the documents the user has access to
+    // fetchDocuments(token, queryParameters, callbck)
+    fetchDocuments(token, {}, (action) => {
```

Comment:
**Code Quality - Nitpick**
If you consider mapping dispatch to action creators, you can directly dispatch the result from the `fetchDocuments` action creator instead of passing it a callback that waits for it to resolve then dispatches directly from the store.